tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AerospikeTests.cs

```diff
@@ -26,72 +29,32 @@ public AerospikeTests(ITestOutputHelper output)
         [MemberData(nameof(PackageVersions.Aerospike), MemberType = typeof(PackageVersions))]
         [Trait("Category", "EndToEnd")]
         [Trait("Category", "ArmUnsupported")]
-        public void SubmitTraces(string packageVersion)
+        public async Task SubmitTraces(string packageVersion)
         {
             using var telemetry = this.ConfigureTelemetry();
             using (var agent = EnvironmentHelper.GetMockAgent())
             using (RunSampleAndWaitForExit(agent, packageVersion: packageVersion))
             {
-                var expectedSpans = new List<string>
-                {
-                    // Synchronous
-                    "Write",
-                    "Write",
-                    "Write",
-                    "Write",
-                    "Read",
-                    "Exists",
-                    "BatchGetArray",
-                    "BatchExistsArray",
-                    "QueryRecord",
```

Comment:
This is the entry that was failing in the more recent server versions - it uses `QueryPartition` instead